src/App.js

```diff
@@ -1,22 +1,30 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Books from './redux/books/books';
+import Categories from './redux/categories/categories';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Router>
+          <nav>
+            <ul>
+              <li><Link to="/">Books</Link></li>
+              <li><Link to="/categories">Categories</Link></li>
+            </ul>
+            <hr />
+          </nav>
+          <Switch>
+            <Route path="/categories"><Categories /></Route>
+            <Route path="/"><Books /></Route>
```

Comment:
- [ ] Also the navbar should be in the component folder.

![nav](https://user-images.githubusercontent.com/61562626/140209593-02fa86d0-3d1e-4f0d-8614-b539abedddf2.jpeg)